base_wamas_ubl/views/wamas_menu.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <menuitem
+        id="menu_wamas_parent"
+        name="WAMAS"
+        parent="base_edi.menu_edi_root"
+        sequence="10"
```

Comment:
I had to increase this one to 100 otherwise it is used as the action for the EDI icon in the home

Reply:
actually not crazy to put a higher value to put it at the end, as this is for debug